docs/articles/documentation/guides/basic-guides/interact-with-the-page.md

```diff
@@ -330,6 +327,45 @@ On touch devices, TestCafe emulates touch events instead of mouse events.
 
 Mouse event | Touch event
 ----------- | -------------
-`mousemove` (when hovering or dragging) | `touchmove` (dragging only)
+`mousemove` (when hovering or dragging) | `touchmove` (when dragging)
```

Comment:
Is it possible to rewrite these without the -ing form?

Reply:
I've been thinking and can't seem to come up with one
@VasilyStrelyaev any thoughts?